starter/js/atm.js

```diff
@@ -1,42 +1,32 @@
-//Begin with the document ready function
-
-    //Checking account deposit function
-
-      //On click of the depositChecking button
-
-        //Get value from the amountChecking input field
-
-        //Take that value and add it to the existing value in the checkingBalance div
-
-    //Checking account withdrawl funtion
-
-      //On click of the withdrawChecking button
-
-        //Get value from the amountChecking input field
-
-        // If that value is greater than the value in the account ignore that action
-        // In other words if this would put the account into a negative balance do not allow it
-
-        //Else subtract that value from the current amount in the checking account
-
-    //Savings account deposit function
-
-      //On click of the depositSavings button
-
-        //Get value from the amountSavings input field
-
-        //Take that value and add it to the existing value in the savingsBalance div
-
-    //Savings account withdraw funtion
-
-      //On click of the withdrawl button
-
-        //Get value from the amountSavings input field
-
-         //If that value is greater than the value in the account ignore that action
-         //In other words if this would put the account into a negative balance do not allow it
-
-         //Else subtract that value from the current amount in the savings account
+$(document).ready(function() {
+  $('#depositChecking').on('click', function() {
+    return ('#amountChecking').val();
+    return amountChecking + checkingBalance
```

Comment:
You can only `return` one thing within a function. Here you want to create a variable to get the deposit value from the input, and then add that to the total (which you also have to grab from the DOM) to get a new 

```js
   // When the checking deposit button is clicked
    $('#despositChecking')on('click', function(event) {
      // Get the deposit amount
      var depositAmount = $('#amountChecking').val();
      // Get the current balance from the div, not an input
      var currentBalance =  $('#checkingBalance').text()
      // Add the two together to get a new balance
      var newBalance = depositAmount + currentBalance;
      // Put the new balance on the page by passing it the new balance
      $('#checkingBalance').text(newBalance)
    });
```